### PR TITLE
Ex4 Submission

### PR DESCRIPTION
**Pull request by tomerach**
Hi @robi-y, @ItamarShDev,

Please review my submission,

Thank you,
Tomer

### CODE REVIEW

File: src/map.js [hunk 1]
```diff
@@ -8,6 +8,6 @@ Map.prototype.find_a_person = function(name) {
 };
 
 Map.prototype.find_location = function(name){
-	return false;
+	return true;
```
Comment (ItamarShDev) [inline src/map.js]: this is not what the function does.
it need to find the locaion

false tests is not accountable

**Comment by ItamarShDev** [comment]
the way i see it' your test does not test a thing 
because you just return true or false on every method
please add some basic login to show that the test really tested something

**Comment by tomerach** [comment]
@ItamarShDev  
Fixed

**Comment by ItamarShDev** [comment]
:+1:

**Pull request closed without merging**